src/components/BioLinkButton/styles.scss

```diff
@@ -1,75 +1,84 @@
 @import "node_modules/@biotope/element/lib/host.mixin";
 @import '../../resources/scss/defaultSassImports.scss';
+
 @include host('bio-link-button') {
     a {
-        $border: 2px;
-        display: block;
-        width: 160px;
-        padding: 10px 0px;
-        background: transparent;
-        color: $bio-black;
-        border: $border solid $bio-orange;
-        border-radius: 50px;
-        font-family: $font-family-titles;
-        font-size: 15px;
-        font-weight: $font-weight-light;
-        text-transform: uppercase;
-        text-decoration: none;
-        text-align: center;
-        transition: 0.15s ease-out;
-        cursor: pointer;
-        &:hover {
-            border: 2px solid transparent;
-            background: $bio-orange;
-            color: $bio-white;
-        }
+		$border: 2px;
+
+		display: block;
+		width: 160px;
+		padding: 10px 0px;
+		background: transparent;
+		color: $bio-black;
+		border: $border solid $bio-orange;
+		border-radius: 50px;
+		font-family: $font-family-titles;
+		font-size: 15px;
+		font-weight: $font-weight-light;
+		text-transform: uppercase;
+		text-decoration: none;
+		text-align: center;
+		transition: 0.15s ease-out;
+		cursor: pointer;
+
+		&:hover {
+			border: 2px solid transparent;
+			background: $bio-orange;
+			color: $bio-white;
+		}
+
         &.blue {
-            border: $border solid $bio-dark-blue;
-            &:hover {
-                background: $bio-dark-blue;
-            }
-        }
-        &.white {
-            border: $border solid $bio-white;
-            color: $bio-white;
-            &:hover {
-                background: $bio-white;
-                color: $bio-orange;
-            }
-        }
-        &.compact {
-            min-width: 130px;
-            @include breakpoint($medium) {
-                min-width: 160px;
-            }
-        }
-        &.whiteText {
-            color: $bio-white;
-        }
+			border: $border solid $bio-dark-blue;
+			
+			&:hover{
```

Comment:
almost perfect :) still missing some spacings on the styles.scss. Please check the whole file.